pkg/component/component.go

```diff
@@ -1069,7 +1091,7 @@ func Update(client *occlient.Client, componentSettings config.LocalConfigInfo, n
 			}
 
 			// Build it
-			err = Build(client, componentName, applicationName, true, stdout, false)
+			//err = Build(client, componentName, applicationName, true, stdout, false)
```

Comment:
TODO: remove this line instead of just commenting it out.
Updating configuration shouldn't trigger rebuild.